templates/repo/home.tmpl

```diff
@@ -83,7 +83,7 @@
 					<a href="{{.Repository.Link}}/find/{{.BranchNameSubURL}}" class="ui compact basic button">{{.locale.Tr "repo.find_file.go_to_file"}}</a>
 				{{end}}
 
-				{{if and .CanWriteCode .IsViewBranch (not .Repository.IsArchived) (not .IsViewFile)}}
+				{{if and .CanWriteCode .IsViewBranch (not .Repository.IsMirror) (not .Repository.IsArchived) (not .IsViewFile)}}
```

Comment:
Is `IsMirror` only true for pull mirrors, or also for push mirrors?
If yes, this will introduce a bug for push mirrors.
Either way, we should fix this inconsistency eventually.

Reply:
I have not tested push mirrors

Reply:
How it could be:

`isMirror` -> either push or pull
`isPushMirror` -> push
`isPullMirror` -> pull

Even better, could be explicit and remove `isMirror` to force the user to think about it. But it might be that customization templates rely on this variable, making the PR at least unsuitable for backport.

Reply:
how do create a push mirror

Reply:
That's a legacy problem. In old ages, there is only "pull mirror", so `IsMirror` means pull mirror.

Later, when adding "push mirror", the name is always "push mirror".

Reply:
> how do create a push mirror

repo settings > repository > mirror settings

Reply:
> That's a legacy problem. In old ages, there was only "pull mirror", so `IsMirror` always means pull mirror.
> 
> Later, when adding "push mirror", the name is always "push mirror".

so 'IsMirror` or `IsPushMirror` both right?

Reply:
this is push mirror:
![image](https://github.com/go-gitea/gitea/assets/89133723/9f8c82c3-8b09-4fb3-a9e7-7db3e4c1b9c2)